CHANGELOG.md

```diff
@@ -0,0 +1,153 @@
+# Changelog
+
+## [1.1](https://github.com/arborx/arborx/tree/v1.1) (2021-09-23)
+
+[Full Changelog](https://github.com/arborx/arborx/compare/v1.0...v1.1)
+
+**New features:**
+
+- Allow k-DOP as a bounding volume in BVH [\#386](https://github.com/arborx/ArborX/pull/386)
+
+**Enhancements:**
+
+- Store a dimension value in a DBSCAN datafile [\#523](https://github.com/arborx/ArborX/pull/523)
+- Enable knn queries with spheres and boxes [\#495](https://github.com/arborx/ArborX/pull/495) and [\#494](https://github.com/arborx/ArborX/pull/494)
+- Use double-precision floating-points to normalize `Ray` direction [\#535](https://github.com/arborx/ArborX/pull/535)
+- Improve performance of the brute force algorithm [\#534](https://github.com/arborx/ArborX/pull/534)
+- Improve performance of the DBSCAN algorithm (new FDBSCAN-DenseBox algorithm) [\#508](https://github.com/arborx/ArborX/pull/508)
+- Allow non-View input for DBSCAN [\#509](https://github.com/arborx/ArborX/pull/509)
+- Install CMake package version file [\#521](https://github.com/arborx/ArborX/pull/521)
+- Add a simple intersection example [\#513](https://github.com/arborx/ArborX/pull/513)
+- Improve performance for the SYCL backend through the use of oneDPL for sorting [\#456](https://github.com/arborx/ArborX/pull/456)
+
+**Fixed bugs:**
+
+- Fix a bug in DBSCAN noise marking [\#525](https://github.com/arborx/ArborX/pull/525)
+
+## [1.0](https://github.com/arborx/arborx/tree/v1.0) (2021-03-13)
+
+[Full Changelog](https://github.com/arborx/arborx/compare/v1.0-rc0...v1.0)
+
+**New features:**
+
+- Allow early termination of a traversal by a thread [\#427](https://github.com/arborx/ArborX/pull/427)
+- Implement DBSCAN clustering algorithm [\#331](https://github.com/arborx/ArborX/pull/331)
+- Implement brute-force algorithm [\#468](https://github.com/arborx/ArborX/pull/468)
+- Add initial ray-tracing support [\#414](https://github.com/arborx/ArborX/pull/414) and [\#461](https://github.com/arborx/ArborX/pull/461)
+
+**Build changes:**
+
+- Require CMake 3.16 [\#486](https://github.com/arborx/ArborX/pull/486)
+
+**Enhancements:**
+
+- Add `KOKKOS_FUNCTION` to `AccessTraits::size()` in View specialization [\#463](https://github.com/arborx/ArborX/pull/463)
+- Non-backward compatible: Change signature of the nearest callback [\#366](https://github.com/arborx/ArborX/pull/366)
```

Comment:
I think that non-backward compatible changes should be in their own section. It's easier for user to know if they need to change their code.

Reply:
Fixed in [`8b565ab` (#548)](https://github.com/arborx/ArborX/pull/548/commits/8b565abc43d398de3d26d7be96157cb992150bba).